e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -137,4 +138,22 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('M23361 Focus remains in the RHS text box', () => {
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        // # Open reply thread (RHS)
+        cy.clickPostCommentIcon();
+
+        // # Add some text to RHS text box
+        cy.get('#reply_textbox').type(MESSAGES.TINY);
+
+        // # Click on Preview
```

Comment:
Should this comment be removed?

Reply:
Oh wait, it's a step from the ticket. In case you're wondering if not showing in your setup, you need to enable it in "Account Settings >> Advanced >> Preview Pre-release Features >> Show markdown preview...".  In Cypress, you may change the preference via API command - `cy.apiSaveShowMarkdownPreviewPreference()`

Reply:
Great! I wouldn't find the preview link without this setting. I've added `cy.apiSaveShowMarkdownPreviewPreference()` in the test. I didn't add this call to `before()` or `beforeEach()` because this setting is specific to this test.

Reply:
Looks good to me, thanks for the update @clarmso!